Makefile

```diff
@@ -5,7 +5,7 @@ BUILD_DIR ?= out
 BINARY_NAME := admin-helper
 RELEASE_DIR ?= release
 
-LDFLAGS := -X main.Version=$(VERSION) -extldflags='-static' -s -w
+LDFLAGS := -X main.Version=$(VERSION) -extldflags='-static' -s -w $(GO_EXTRA_LDFLAGS)
```

Comment:
Wondering why you choose the wording 'extra'

Reply:
Reused some naming used in some Makefiles I was looking at (`oc` and/or `docker`), I can use another name if you want. `GO_LDFLAGS` I assume?

Reply:
I switched to GO_BUILDFLAGS/GO_LDFLAGS.